src/components/Header/web3/NetworkModal/index.tsx

```diff
@@ -33,40 +82,188 @@ export default function NetworkModal({
   customToggleModal?: () => void
   disabledMsg?: string
 }): JSX.Element | null {
+  const theme = useTheme()
   const { isWrongNetwork } = useActiveWeb3React()
+  const [requestSaveProfile] = useUpdateProfileMutation()
+  const { userInfo } = useSessionInfo()
+
+  const [favoriteChains, setFavoriteChains] = useState<string[]>(userInfo?.data?.favouriteChainIds || [])
+  const [isEdittingMobile, setIsEdittingMobile] = useState(false)
+  const wrapperRef = useRef<HTMLDivElement>(null)
+
   const networkModalOpen = useModalOpen(ApplicationModal.NETWORK)
   const toggleNetworkModalGlobal = useNetworkModalToggle()
   const toggleNetworkModal = customToggleModal || toggleNetworkModalGlobal
+
+  const droppableRefs = useRef<HTMLDivElement[]>([])
+  const { supportedChains } = useChainsConfig()
+
+  const handleDrop = (chainId: string, dropId: string) => {
+    if (dropId === FAVORITE_DROPZONE_ID) {
+      const chainInfo = supportedChains.find(item => item.chainId.toString() === chainId)
+      if (chainInfo && !favoriteChains.includes(chainInfo)) {
+        saveFavoriteChains([...favoriteChains, chainInfo.chainId.toString()])
+      }
+    }
+    if (dropId === CHAINS_DROPZONE_ID) {
```

Comment:
if(dropId === CHAINS_DROPZONE_ID&&(dropId === CHAINS_DROPZONE_ID))...

Reply:
I don't get this

Reply:
if(dropId === CHAINS_DROPZONE_ID &&  favoriteChains.some(fChainId => fChainId === chainId) ){}